### PR TITLE
Fix lint

### PR DESCRIPTION
**Pull request by algrs**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -39,6 +39,10 @@ task :test do
   raise unless system "nose2"
 end
 
+task :unittest do
```
Comment (algrs) [inline Rakefile]: Just because I find myself typing `rake unittest` half the time and `rake test` half the time....

File: baiji/connection.py [hunk 2]
```diff
@@ -421,7 +421,7 @@ def etag_matches(self, key_or_file, other_etag):
             # print "  - possible_part_sizes:", possible_part_sizes
             possible_part_sizes = set([part_size for part_size in possible_part_sizes if part_size <= max_part_size and part_size >= 1024*1024*5])
             # print "  - possible_part_sizes:", possible_part_sizes
-            if len(possible_part_sizes) == 0:
+            if not possible_part_sizes:
```
Comment (algrs) [inline baiji/connection.py]: The pythonic way; empty lists are falsey. Newly enforced as of pylint 1.7.

File: baiji/copy.py [hunk 3]
```diff
@@ -86,7 +86,7 @@ def __init__(self, src, dst, connection):
             raise ValueError("{} is a directory (not copied; use recursive mode to copy directories".format(self.src.raw))
 
         if self.dst.isdir:
-            self.dst.path = os.path.join(self.dst.path, os.path.basename(self.src.path))
+            self.dst.path = path.join(self.dst.path, path.basename(self.src.path))
```
Comment (algrs) [inline baiji/copy.py]: We were importing baiji's path and then using os's...

File: baiji/util/lookup.py [hunk 4]
```diff
@@ -13,5 +13,5 @@ def get_versioned_key_remote(bucket, key_name, version_id=None):
         if e.status == 400:
             raise InvalidVersionID("Invalid versionID %s" % version_id)
         else:
-            raise e
```
Comment (algrs) [inline baiji/util/lookup.py]: When in an except block, to reraise the original exception just say `raise`; trying `raise e` causes the exception handler to throw an exception, which is hard to debug...

**Comment by chenliu0831** [comment]
👍

**Pull request merged**